### PR TITLE
Fix walkabout

### LINKED ISSUE #666: Fix walkabout
The issue currently is that the players' details (their former force, character, etc) are only entered into the global table when the token is called (after the timeout) and so if someone leaves and returns within the time of their timeout, their details are lost since `clean_on_join` removes their walkabout status.

Needless to say: I want to change a lot about the way I "fixed" things then. Why did I make a second table in `global`? Only past me knows.

### PR DESCRIPTION
**Pull request by mheguy**
Not returning player from walkabout if they rejoin before timer expired, update refs to global.walking

Closes #666

### CODE REVIEW

**Comment by grilledham** [comment]
Not tempted to move the global variables into Global?

**Comment by mheguy** [comment]
Oddly enough: hadn't thought about it and just kinda took it as it was. But yeah that would make a lot more sense.

File: features/redmew_commands.lua [hunk 1]
```diff
@@ -38,7 +38,7 @@ local function kill(args, player)
         end
     end
 
-    if global.walking and ((player and global.walking[player.index]) or (target and global.walking[target.index])) then
+    if global.walkabout and ((player and global.walkabout[player.index]) or (target and global.walkabout[target.index])) then
```
Comment (grilledham) [inline features/redmew_commands.lua]: Didn't these get moved into Global?

**Review by grilledham — DISMISSED** [review]

**Review by linaori — APPROVED** [review]
Changes look alright, I've never used the function itself so I can't really comment on that

**Review by grilledham — APPROVED** [review]

**Pull request merged**